graylog2-web-interface/src/logic/rest/FetchProvider.ts

```diff
@@ -191,8 +204,11 @@ export class Builder {
   }
 
   build() {
+    // eslint-disable-next-line no-nested-ternary
     const headers: RequestHeaders = this.body
-      ? { ...this.options, 'Content-Type': this.body.mimeType }
+      ? this.body.mimeType
```

Comment:
I think we can slightly simplify this block as:

```
    const headers: RequestHeaders = this.body && this.body.mimeType
        ? { ...this.options, 'Content-Type': this.body.mimeType }
        : this.options;
```

It would also let us get rid of the eslint disable comment

Reply:
Thanks! Fixed it straight away. :grin: